zerver/migrations/0271_huddle_set_recipient_column_values.py

```diff
@@ -1,20 +0,0 @@
-# -*- coding: utf-8 -*-
-from django.db import migrations
-
-
-class Migration(migrations.Migration):
-
-    dependencies = [
-        ('zerver', '0270_huddle_recipient'),
-    ]
-
-    operations = [
-        migrations.RunSQL(
-            """
-            UPDATE zerver_huddle
-            SET recipient_id = zerver_recipient.id
-            FROM zerver_recipient
-            WHERE zerver_recipient.type_id = zerver_huddle.id AND zerver_recipient.type = 3;
-            """,
-            reverse_sql='UPDATE zerver_huddle SET recipient_id = NULL'),
-    ]
```

Comment:
You seem to be accidentally deleting my migration :cry: `tools/renumber-migrations` can be useful for dealing with the hassle of renumbering, when doing a rebase on master.

Reply:
sorryyyy

Reply:
To undo this I just need to copy this file, run `tools/renumber-migrations` and then `makemigrations`?

Reply:
I guess that would work :thinking: What I'd do personally would be:
`git reset --soft` to the base of your branch -> revert the action of deleting the 0271 migration -> remove the migration you've added -> `manage.py makemigrations` to have django take care of things -> stage what's needed and commit (you'll need to squash anyway at some point)

Though now that I typed this out, maybe that's more work lol